app/controllers/project_stats_controller.rb

```diff
@@ -288,7 +288,7 @@ def daily_activity
         data: series_dataset
       }
       # Calculate moving average over ndays
-      series_counts = stat_data.map { |e| e[desired_field] }
+      series_counts = stat_data.pluck(desired_field)
```

Comment:
Does pluck really return the *series* in this case?

Reply:
Pluck is implemented on Enumerable as well as on a Relation. https://docs.rubocop.org/rubocop-rails/cops_rails.html#railspluck I believe it should be the same, but we could write a small test to make sure if that's not already covered.

Reply:
I added a test to test the values of the averages https://github.com/coreinfrastructure/best-practices-badge/pull/1893/commits/ddb96b432201b99d742344c56d43dc7fc9192bdb#diff-d9a50e554f50eae4a07564b1b88ae0977f17de3ee528054b343abc305395f9edR198